release/sbin/clear-all.sh

```diff
@@ -48,8 +60,15 @@ else
       rm_dir "$host" "$dir"
     else
       echo "clear tablet data and log in $dir with endpoint $host:$port "
-      cmd="cd $dir && rm -rf recycle db logs"
-      run_auto "$host" "$cmd"
+      for item in "${dirname[@]}"
+      do
+        if [[ $item == "/" ]]; then
```

Comment:
there are many cases elsewhere like `/usr`, we can't hard check it. 

If the array `dirname` is constructed corrected, simply make the cmd as
```bash
cmd="cd $dir && rm -rf \"$item\""
```

Reply:
`/` is protected by `preserve-root` option default.